lib/qesapdeployment.pm

```diff
@@ -480,4 +485,50 @@ sub qesap_ansible_script_output {
     return $output;
 }
 
+=head3 qesap_create_aws_credentials
+
+    Creates a AWS credentials file as required by QE-SAP Terraform deployment code.
+=cut
+
+sub qesap_create_aws_credentials {
+    my ($key, $secret) = @_;
+    my %paths = qesap_get_file_paths();
+    my $credfile = script_output q|awk -F\" '/aws_credentials/ {print $2}' | . $paths{qesap_conf_trgt};
+    save_tmp_file('credentials', join(' ', "[default]\naws_access_key_id", '=', $key,
```

Comment:
I propose to change it in `join("\n"` and code each file line for each join element. It is to keep the code more readable.

Reply:
That way the file will lack a newline at the end, which I guess it could be non-essential. I can change it to:
```
save_tmp_file('credentials', join("[default]\naws_access_key_id = $key\naws_secret_access_key = $secret\n"));
```
If you feel it's more readable.

Reply:
Probably not

Reply:
I changed it anyway. Liked it more after the fact :P